src/mudlet.cpp

```diff
@@ -1257,7 +1257,19 @@ bool mudlet::setLabelClickCallback( Host * pHost, const QString & name, const QS
     QMap<QString, TLabel *> & labelMap = mHostLabelMap[pHost];
     if( labelMap.contains( name ) )
     {
-        labelMap[name]->setScript( pHost, func, pA );
+        labelMap[name]->setClick( pHost, func, pA );
```

Comment:
Note, I heavily edited my previous comment as I thought the `TLabel` methods had return values - I was wrong and I re-targetted my comments to the `mudlet` methods so that the `TLuaInterpreter` ones return the true or false from the `mudlet` ones instead of blindly returning true whatever the `mudlet` methods return.